src/renderer/components/RecentConnections.tsx

```diff
@@ -32,54 +55,42 @@ const RecentConnections = () => {
       <div className="recent-wrapper">
         <h1>Recent Connections</h1>
         <div className="d-flex justify-content-center">
-          <Table className="table">
-            <thead>
-              <tr>
-                <th>Nick Name</th>
-                <th>Database Type</th>
-                <th>Address</th>
-                <th>Port</th>
-                <th>User Name</th>
-                <th>Actions</th>
-              </tr>
-            </thead>
-            <tbody>
-              {connect.map((value) => {
-                if (value.connectionConfig.config === 'manual') {
+          <form onSubmit={submitForm}>
```

Comment:
If possible, can you try to use the React-Bootstrap elements instead of basic HTML? It provides a bit more styling out of the box and will make everything else look more consistent. ie here you would use \<Form\> imported from react-bootstrap

https://react-bootstrap.github.io/getting-started/introduction/